### DISCUSSION TITLE
Question: Transparency Mandates; Intended Use of DRG code

### DISCUSSION THREAD

**Discussion by colber35** [CMSgov/price-transparency-guide#504] [Q&A] [answered]
In the list of 500 CMS services identified for Transparency Yr 2 deliverable, there are four descriptions that were listed with DRGs vs. CPTs. 

1) What is the intent to get a grouped price of multiple procedures that make up a complex procedure? For example, spinal fusion anterior DRG Code 473 vs. the single price for the spinal fusion anterior CPT Code 22899?
2) Would insurers be compliant to return a MLE estimate for spinal fusion anterior based on CPT 22899?
3) Would insurers be non-compliant to return a MLE estimate for a spinal fusion anterior based on CPT vs DRG if the estimate of the member liability would be different, and if yes by how much of a difference would be non-compliant?

**Comment by shaselton-usds** [comment, marked as answer]
Closing this thread due to it being about the tool vs the MRFs. You can find a list of the current 500 items/services here: https://www.cms.gov/healthplan-price-transparency/resources/500-items-services
